venia-integration-tests/src/tests/integrationTests/multiStore/storeSwitching.spec.js

```diff
@@ -230,6 +223,61 @@ const interceptAccessoriesPageRequests = expectedStoreCode => {
     }).as('getMockFilterInput');
 };
 
+/**
+ * Setup network intercepts that return mocked data for requests
+ * specific to the tops category page
+ *
+ * @param {String} expectedStoreCode
+ */
+const interceptTopsPageRequests = expectedStoreCode => {
+    cy.intercept(
+        'GET',
+        getCategoriesCall,
+        getInterceptHandler('topsCategory', expectedStoreCode)
+    ).as('getMockCategories');
+    cy.intercept(
+        'GET',
+        getBreadcrumbsCall,
+        getInterceptHandler('topsCategory', expectedStoreCode)
+    ).as('getMockBreadcrumbs');
+    cy.intercept(
+        'GET',
+        getCategoryDataCall,
+        getInterceptHandler('topsCategory', expectedStoreCode)
+    ).as('getMockCategoryData');
+
+    cy.intercept('GET', getProductFiltersByCategoryCall, req => {
+        expect(req.headers.store).to.equal(expectedStoreCode);
+        req.reply({
+            fixture: `${DATA_DIRECTORY}/default/topsProductFilters.json`
+        });
+    }).as('getMockProductFilter');
+    cy.intercept('GET', getFilterInputsForCategoryCall, req => {
+        expect(req.headers.store).to.equal(expectedStoreCode);
+        req.reply({
+            fixture: `${DATA_DIRECTORY}/default/topsFilterInputs.json`
+        });
+    }).as('getMockFilterInput');
+};
+
+/**
+ * Setup network intercepts that return mocked data for requests
+ * specific to the product 1 product page
+ *
+ * @param {String} expectedStoreCode
+ */
+const interceptProduct1PageRequests = expectedStoreCode => {
```

Comment:
Same thing from above comment, see if we can reduce some code by adding a second parameter like `expectedProduct`

Reply:
I've reworked the code and simplified things a bit like you suggested.